hack/unwanted-dependencies.json

```diff
@@ -28,9 +28,11 @@
       "github.com/gorilla/mux": "unmaintained, archive mode",
       "github.com/gorilla/rpc": "unmaintained, archive mode",
       "github.com/gorilla/schema": "unmaintained, archive mode",
-      "github.com/gorilla/websocket": "unmaintained, archive mode",
+<<<<<<< HEAD
       "github.com/gregjones/httpcache": "unmaintained, archive mode",
       "github.com/grpc-ecosystem/go-grpc-prometheus": "unmaintained, archive mode",
+=======
+>>>>>>> e90bd1d2eba (WebSocket Client and V5 RemoteCommand Subprotocol)
```

Comment:
@seans3 you have some rebase problems here 😄

Reply:
Fixed

Reply:
is still present in this commit

Reply:
After the rebase, this file is no longer a part of the PR; so I think it's fixed. Please let me know if it isn't.